generated/browser/js/app.js

```diff
@@ -12,6 +12,14 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     });
 });
 
+// This app.run is for listening to errors broadcasted by ui-router, usually originating from resolves
+app.run(function ($rootScope) {
+    $rootScope.$on('$stateChangeError', function (event, toState, toParams, fromState, fromParams, thrownError) {
+        console.info('The following error was thrown by ui-router. The origin of this error is probably a resolve function:');
```

Comment:
Might we also want to include the `toState` and/or `fromState` in an error message? Or is that included in the `thrownError` itself and therefore redundant?